### PR TITLE
CON-16: Test plan

### PR DESCRIPTION
**Pull request by benjaoming**
## Description

Originally, I wrote a test plan in https://github.com/GovStackWorkingGroup/bb-consent/pull/13 but had issues getting anyone to review it.

I think we should write one that we're all onboard with and share it with SolDevelo.

## Related Issue

https://govstack-global.atlassian.net/browse/CON-16

### CODE REVIEW

**Comment by benjaoming** [comment]
I haven't actually seen a test plan in another BB, but to summarize on recent discussions, ideas and priorities, I think this could be a good candidate.

**Comment by benjaoming** [comment]
CC: @aaviksoo

**Review by benjaoming — COMMENTED** [review]
Approved during meeting 2023-07-16

File: test/plan.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ Those files are available here:
 
 Please see the Development section further down for implementation details.
 
+### Proposing a new scenario
```
Comment (aaviksoo) [inline test/plan.md:31]: Sounds good to me

**Pull request merged**